moneyed/constants.py

```diff
@@ -0,0 +1,398 @@
+    'MYR': '458',
+    'BOB': '068',
+    'BOV': '984',
+    'BRL': '986',
+    'BSD': '044',
+    'BTN': '064',
+    'NOK': '578',
+    'BWP': '072',
+    'ZAR': '710',
+    'BYN': '933',
+    'BZD': '084',
+    'CAD': '124',
+    'CDF': '976',
+    'XAF': '950',
+    'CHF': '756',
+    'CHE': '947',
+    'CHW': '948',
+    'NZD': '554',
+    'CLP': '152',
+    'CLF': '990',
+    'CNY': '156',
+    'CNH': 'Nil',
+    'COP': '170',
+    'COU': '970',
+    'CRC': '188',
+    'CUP': '192',
+    'CUC': '931',
+    'CVE': '132',
+    'CZK': '203',
+    'DJF': '262',
+    'DKK': '208',
+    'DOP': '214',
+    'DZD': '012',
+    'EGP': '818',
+    'MAD': '504',
+    'ERN': '232',
+    'ETB': '230',
+    'FJD': '242',
+    'FKP': '238',
+    'JPY': '392',
+    'GBP': '826',
+    'GEL': '981',
+    'GHS': '936',
+    'GIP': '292',
+    'GMD': '270',
+    'GNF': '324',
+    'GTQ': '320',
+    'GYD': '328',
+    'HKD': '344',
+    'HNL': '340',
+    'HRK': '191',
+    'HTG': '332',
+    'HUF': '348',
+    'IDR': '360',
+    'ILS': '376',
+    'IQD': '368',
+    'IRR': '364',
+    'ISK': '352',
+    'JMD': '388',
+    'JOD': '400',
+    'KES': '404',
+    'KGS': '417',
+    'KHR': '116',
+    'KMF': '174',
+    'KPW': '408',
+    'KRW': '410',
+    'KWD': '414',
+    'KYD': '136',
+    'KZT': '398',
+    'LAK': '418',
+    'LBP': '422',
+    'LKR': '144',
+    'LRD': '430',
+    'LSL': '426',
+    'LYD': '434',
+    'MDL': '498',
+    'MGA': '969',
+    'MKD': '807',
+    'MMK': '104',
+    'MNT': '496',
+    'MOP': '446',
+    'MRU': '929',
+    'MUR': '480',
+    'MVR': '462',
+    'MWK': '454',
+    'MXN': '484',
+    'MXV': '979',
+    'MZN': '943',
+    'NAD': '516',
+    'XPF': '953',
+    'NGN': '566',
+    'NIO': '558',
+    'SEK': '752',
+    'NPR': '524',
+    'OMR': '512',
+    'PAB': '590',
+    'PEN': '604',
+    'PGK': '598',
+    'PHP': '608',
+    'PLN': '985',
+    'PYG': '600',
+    'QAR': '634',
+    'RON': '946',
+    'RSD': '941',
+    'RUB': '643',
+    'RWF': '646',
+    'SAR': '682',
+    'SBD': '090',
+    'SCR': '690',
+    'SDG': '938',
+    'SGD': '702',
+    'SLL': '694',
+    'SOS': '706',
+    'SRD': '968',
+    'SSP': '728',
+    'STN': '930',
+    'SVC': '222',
+    'SYP': '760',
+    'SZL': '748',
+    'THB': '764',
+    'TJS': '972',
+    'TMT': '934',
+    'TND': '788',
+    'TOP': '776',
+    'TRY': '949',
+    'TTD': '780',
+    'TWD': '901',
+    'TZS': '834',
+    'UAH': '980',
+    'UGX': '800',
+    'USN': '997',
+    'UYU': '858',
+    'UYI': '940',
+    'UYW': '927',
+    'UZS': '860',
+    'VES': '928',
+    'VND': '704',
+    'VUV': '548',
+    'WST': '882',
+    'YER': '886',
+    'ZMW': '967',
+}
+
+XCOMMON_ISO_CODES = {
+    'XAG': '961',
+    'XAU': '959',
+    'XBA': '955',
+    'XBB': '956',
+    'XBC': '957',
+    'XBD': '958',
+    'XDR': '960',
+    'XPD': '964',
+    'XPT': '962',
+    'XSU': '994',
+    'XTS': '963',
+    'XUA': '965',
+    'XXX': '999',
+}
+
+OBSELETE_ISO_CODES = {
```

Comment:
@pooyamb Could you let me know where you got this info from i.e. numeric codes of these obsolete currencies? I'm trying to fill the gaps between the large list of currencies that Babel has, but we don't have, since we are starting with ISO 4217

Reply:
@spookylukey Yes these are extracted from the babel(cldr) itself, babel doesn't provide a direct way to get the obslete currencies directly itself, but it has some notions like a date range added to those currencies for example "Yugoslavian Convertible Dinar (1990–1992)" in locale.currencies for "en_US". I think I wrote a script to extract those and checked all the currencies myself one by one.
BTW the main problem with these is that these are not guaranteed to be available for all locales, for example "YUD" is available in "en_US" locale but is missing in "fa_IR".